content/02.main-text.md

```diff
@@ -1,3 +1,21 @@
 ## Main text {.page_break_before}
 
-*Work in progress*
+### Introduction
+
+Open research – which includes sharing code, data, and manuscripts – benefits the researchers who engage in open practices [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+`TODO: more references needed`
+Here we advocate for the benefits of writing review articles openly, specifically planning, organizing, writing, and editing this type of manuscript collaboratively in an unrestricted public forum.
+Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
+In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by ensuring that it represents diverse opinions and does not neglect relevant research.
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately.
+In addition, discussing manuscripts in the open provides one form of pre- and post-publication peer review `TODO: define this or provide a reference?`, incentivizing it with potential manuscript authorship.
+However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
+
+We present potential solutions to these challenges based on our recent experience leading a collaborative review "Opportunities And Obstacles For Deep Learning In Biology And Medicine" [@doi:10.1101/142760].
+`TODO: and the SciHub manuscript, or add that at the end to show how these practices generalize?`
+Our review attracted 27 authors from 20 different institutions who were not determined in advance.
+`TODO: confirm institution count`
+We wrote entirely in the open without restrictions on who was welcome to contribute.
+Although we recruited some authors for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
```

Comment:
`recruited` is pretty strong. Maybe `requested that some authors participate for their specific expertise...`

There were people that I asked that did not end up contributing.